onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -915,7 +915,13 @@ TEST(PoolTest, AveragePool_DefaultStrides) {
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
 }
 
+#ifdef USE_QNN
+// TODO: Unskip when fixed AB#14594
+// Error message: ComposeGraph Failed to compose Qnn graph.
+TEST(PoolTest, DISABLED_AveragePool_10_ceil1_2d) {
```

Comment:
Thanks for this PR. I have a pr that fixes this test: https://github.com/microsoft/onnxruntime/pull/15477

I'll extract the fix and make a separate/smaller PR that we can merge after this one.